src/vaibhav/MathematicalOperationWhileLoop.java

```diff
@@ -0,0 +1,52 @@
+package vaibhav;
+
+public class MathematicalOperationWhileLoop {
+	
+	void displayEvenNumbers(int start,int end) {
+		int i=start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?